tools/rack/rack.plugin/src/com/ge/research/rack/IngestInstanceDataHandler.java

```diff
@@ -92,7 +91,7 @@ private IngestionStatus uploadModelFromYAML(String yamlPath, IProgressMonitor mo
         }
 
         if (dedupSteps.contains(yamlPath)) {
-            RackConsole.getConsole().warning("Skipping previously executed step at: " + yamlPath);
+            ErrorMessageUtil.warning("Skipping previously executed step at: " + yamlPath);
```

Comment:
I grep'ed both RackConsole and ErrorMessageUtil in a checkout of the dev branch and found some files still use RackConsole (in fact, more files than the files using ErrorMessageUtil).  Are source files supposed to use one or the other consistently, or what is the rule picking one or the other?